DESCRIPTION

```diff
@@ -19,4 +19,5 @@ Suggests:
 BugReports: https://github.com/Bioconductor/BiocManager/issues
 VignetteBuilder: knitr
 License: Artistic-2.0
-RoxygenNote: 6.0.1
+RoxygenNote: 6.1.0
+Encoding: UTF-8
```

Comment:
Was wondering why this is introduced? I don't think there should be UTF-8 characters in our files?

Reply:
I was getting a warning message from `roxygen`